packages/types/src/displayConfig.ts

```diff
@@ -2,7 +2,7 @@ import type { DisplayThemeJSON } from './json';
 import type { ClerkResource } from './resource';
 
 export type PreferredSignInStrategy = 'password' | 'otp';
-export type CaptchaWidgetType = 'smart' | 'always_visible' | 'invisible' | null;
```

Comment:
❓ Did we drop this option? If so, shouldn't we mention it in changelog that the `always_visible` is removed?

Reply:
since it wasn't introduced publicly, you are correct, we shouldn't add it in the changelog. nice!